lib/streamlit/runtime/runtime.py

```diff
@@ -541,9 +532,12 @@ async def _loop_coroutine(self) -> None:
                     return_when=asyncio.FIRST_COMPLETED,
                 )
 
-            # Shut down all AppSessions
-            for session_info in list(self._session_info_by_id.values()):
-                session_info.session.shutdown()
+            # Shut down all AppSessions.
+            # NOTE: We want to fully shut down sessions when the runtime stops for now,
+            # but this may change in the future if/when our notion of a session is no
+            # longer so tightly coupled to a browser tab.
+            for session_info in self._session_mgr.list_active_sessions():
```

Comment:
`list_active_sessions()` implies that there are non-active sessions that are not being shut down here. Is that the case and is it intended?